File: liqp-junit/src/main/kotlin/liqp/Mocks.java
package liqp;

import static java.util.Collections.emptyMap;

import liqp.LiquidRenderer;
import liqp.LiquidParser;
import liqp.nodes.RenderContext;

public class Mocks {
  public static RenderContext mockRenderContext() {
    return new RenderContext(emptyMap(),
          new LiquidParser(),
          new LiquidRenderer());
  }
}

File: liqp-junit/src/main/kotlin/liqp/Mocks.kt
package liqp

import java.util.Collections.emptyMap

import liqp.LiquidRenderer
import liqp.LiquidParser
import liqp.nodes.RenderContext

class Mocks {
  companion object {
    @JvmStatic
    fun mockRenderContext(): RenderContext {
      val parser = provider.createParser()

      return RenderContext(emptyMap<Any, Any>(),
          logic = StrictLogic(),
          parser = parser,
          renderer = parser.toRenderer())
    }
  }

}
